.github/workflows/CI.yaml

```diff
@@ -0,0 +1,30 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up JDK 11
```

Comment:
Consider enabling Gradle cache here (I know it's weird to be part of `setup-java`, but that's where it is 😆